openpype/settings/defaults/system_settings/tools.json

```diff
@@ -1,90 +1,403 @@
 {
     "tool_groups": {
+        "htoa": {
+            "environment": {
+                "HOUDINI_PATH": [
+                    "{STUDIO_SW}/APP/HTOA/{HTOA_VERSION}/HOUDINI{HOUDINI_VERSION}/WINDOWS/htoa-6.1.3.3_rdb15014_houdini-{HTOA_VERSION}",
```

Comment:
If we're mapping all of these to `{STUDIO_SW}` as an example should we also have an example default `STUDIO_SW` defined in the global env?

Reply:
That we should definitely do

Reply:
agreed